Formula/gdu.rb

```diff
@@ -0,0 +1,18 @@
+class Gdu < Formula
+  desc "Disk usage analyzer with console interface written in Go"
+  homepage "https://github.com/dundee/gdu"
+  version "2.2.0"
+  url "https://github.com/dundee/gdu/archive/v#{version}.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/dundee/gdu/archive/v2.2.0.tar.gz"
```
The version is automatically parsed from the URL so there is no need to specify it.

Reply:
Cool, thanks.